specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/virtualMachine.json

```diff
@@ -3576,6 +3580,29 @@
         }
       },
       "description": "Describes the properties of the last installed patch summary."
+    },
+    "Placement": {
+      "properties": {
+        "zonePlacementPolicy": {
+          "type": "string",
+          "description": "Specifies policy for auto zone placement."
+        },
+        "includeZones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of zones to include for auto zone placement."
+        },
+        "excludeZones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of zones to exclude for auto zone placement."
+        }
+      },
+      "description": "The virtual machine automatic zone placement feature."
```

Comment:
Described the user-defined constraints for virtual machine placement on hardware.